### PR TITLE
Requesting code review

### PR DESCRIPTION
**Pull request by Amrhub**
Please be aware that I have committed twice index.html & README.md because I forgot some things and fixed them.

### CODE REVIEW

**Comment by Amrhub** [comment]
Things i forgot to add in first commit: 
1- In indedx.html I forgot to link the stylesheet before committing so had to re-commit 
2-README.md I have had forgotten to remove/replace some of your instructions

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (SajjadAhmad14) [inline index.html:1]: Please add code linter from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) config.
Pull Request should have a short summary describing introduced changes in more detail.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,27 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
```
Comment (SajjadAhmad14) [inline README.md:3]: Please add steps to run the project locally in README.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (SajjadAhmad14) [inline .gitignore]: Please add `node_modules` to gitignore.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Amr Abdelrahem Abdelrahman Ahmed, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by karvel-code — APPROVED** [review]
Hi @Amrhub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**